notificationClient/notificationClient_test.go

```diff
@@ -0,0 +1,85 @@
+package notificationClient_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	e3dbClients "github.com/tozny/e3db-clients-go"
+	"github.com/tozny/e3db-clients-go/accountClient"
+	"github.com/tozny/e3db-clients-go/notificationClient"
+	"github.com/tozny/e3db-clients-go/test"
+	"github.com/tozny/utils-go"
+)
+
+var (
+	e3dbAuthHost          = utils.MustGetenv("E3DB_AUTH_SERVICE_HOST")
+	e3dbAccountHost       = utils.MustGetenv("E3DB_ACCOUNT_SERVICE_HOST")
+	toznyNotificationHost = utils.MustGetenv("TOZNY_NOTIFICATION_SERVICE_HOST")
+	e3dbAPIKey            = utils.MustGetenv("E3DB_API_KEY_ID")
+	e3dbAPISecret         = utils.MustGetenv("E3DB_API_KEY_SECRET")
+	ValidClientConfig     = e3dbClients.ClientConfig{
+		APIKey:    e3dbAPIKey,
+		APISecret: e3dbAPISecret,
+		Host:      toznyNotificationHost,
```

Comment:
for all external endpoints a client should use CYCLOPS_SERVICE_HOST (this will ensure the request is authenticated and the auth n header will be set)